### ISSUE TITLE
Rank-synchronized random numgenerators are out of sync.

### ISSUE THREAD

**Issue by never-to-never** [nest/nest-simulator#2555]
Why can't different processes set different random number seeds during multi-process simulation?

**Comment by heplesser** [comment]
NEST takes care of proper seeding and management of all necessary random number streams in parallel simulations. Please see https://nest-simulator.readthedocs.io/en/v3.3/guides/random_numbers.html#random-internals for details on random numbers in NEST. 

I will close this issue now. If you should get an error message about RNGs out of sync, please create a new issue, include details on the NEST version used and a minimal reproducer. Preferably, use our [template for bug reports](https://github.com/nest/nest-simulator/issues/new?assignees=&labels=&template=bug_report.md&title=).
